tests/TestCase/I18n/NumberTest.php

```diff
@@ -538,4 +538,22 @@ public function testReadableSizeLocalized()
         $result = $this->Number->toReadableSize(512.05 * 1024 * 1024 * 1024);
         $this->assertEquals('512,05 GB', $result);
     }
+
+    /**
+     * test persisting formatter attributes
+     *
+     * @return void
+     */
+    public function testPersistingFormatterAttributes()
+    {
+        Number::formatter([
+            'type' => 'currency',
+            'locale' => 'en_IN',
+            'pattern' => '¤ #,##,##0',
+            'persistOptions' => true
+        ]);
```

Comment:
config() sounds like a good idea to me. Gives nice symmetry with other parts of the framework.